### ISSUE TITLE
Padding settings are ignored for fields in an XML stream.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#10]
```
Padding settings are ignored for fields in an XML stream.

```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 15 Jul 2011 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
Fixed for release 1.1.1.
```

Original comment by `kevin.s...@gmail.com` on 17 Jul 2011 at 11:59
- Changed state: **Fixed**
- Added labels: **Milestone-1.0.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 18 Jul 2011 at 12:00
- Added labels: **Milestone-1.1.1**
- Removed labels: **Milestone-1.0.1**
